src/PQDashboard/Scripts/Default.js

```diff
@@ -482,14 +482,90 @@ function populateFaultsDivWithGrid(data) {
         columns.push({ field: 'thefaulttype', headerText: 'Type', headerStyle: 'width:  6%', bodyStyle: 'width:  6%; height: 20px', sortable: true });
+
+        columns.push({
+            field: 'TreeFaultResistance', headerText: 'Calc. Cause', headerStyle: 'width:  10%', bodyStyle: 'width:  6%; height: 20px', sortable: true,
+            content: function (row, options, td) {
+                let cause = "unknown";
+                let highFound = false;
+                let medFound = false;
+
+                if (row.LowPrefaultCurrentRatio != undefined) {
+                    if (row.LowPrefaultCurrentRatio <= 0.1) {
+                        highFound = true;
+                        cause = "Break";
+                    } else if (row.LowPrefaultCurrentRatio <= 0.5) {
+                        medFound = true;
+                        cause = "Break??";
+                    }
+                }
+                if (row.LightningMilliseconds != undefined) {
+                    if (row.LightningMilliseconds <= 2) {
+                        if (highFound)
+                            return cause + "?";
+                        highFound = true;
+                        cause = "Lightning";
+                    } else if (!highFound && !medFound) {
+                        medFound = true;
+                        cause = "Lightning??";
+                    }
+                }
+                if (row.TreeFaultResistance != undefined) {
+                    if (row.TreeFaultResistance > 20) {
+                        if (highFound)
+                            return cause + "?";
+                        highFound = true;
+                        cause = "Tree";
+                    } else if (row.TreeFaultResistance > 10 && !highFound && !medFound) {
+                        medFound = true;
+                        cause = "Tree??";
+                    }
+                }
+                if (row.GroundCurrentRatio != undefined) {
+                    if (row.GroundCurrentRatio <= 0.1) {
+                        if (highFound)
+                            return cause + "?";
+                        highFound = true;
+                        cause = "Slap/Debris";
+                    } else if (row.GroundCurrentRatio <= 0.5 && !highFound && !medFound) {
+                        medFound = true;
+                        cause = "Slap/Debris??";
+                    }
+                }
+                if (row.PrefaultThirdHarmonic != undefined) {
+                    if (row.PrefaultThirdHarmonic > 0.3) {
+                        if (highFound)
+                            return cause + "?";
+                        highFound = true;
+                        cause = "Arrestor";
+                    } else if (row.PrefaultThirdHarmonic > 0.2 && !highFound && !medFound) {
+                        medFound = true;
+                        cause = "Arrestor??";
```

Comment:
I guess there was a typo in the graphic we used for our presentations. This is referring to a lightning arrester, which is spelled with an `e` rather than an `o`.

```suggestion
                        cause = "Arrester";
                    } else if (row.PrefaultThirdHarmonic > 0.2 && !highFound && !medFound) {
                        medFound = true;
                        cause = "Arrester??";
```